zhmccli/_cmd_partition.py

```diff
@@ -703,3 +741,35 @@ def cmd_partition_console(cmd_ctx, cpc_name, partition_name, options):
         part_console(cmd_ctx.session, partition, refresh, logger)
     except zhmcclient.Error as exc:
         raise click.ClickException("%s: %s" % (exc.__class__.__name__, exc))
+
+
+def cmd_partition_mount_iso(cmd_ctx, cpc_name, partition_name, options):
+
+    client = zhmcclient.Client(cmd_ctx.session)
+    partition = find_partition(cmd_ctx, client, cpc_name, partition_name)
+
+    image_file = options['imagefile']
+    image_fp = open(image_file, 'rb')
+    path, image_name = os.path.split(image_file)
+    partition.mount_iso_image(image_fp, image_name, options['imageinsfile'])
+
+    cmd_ctx.spinner.stop()
+    click.echo('ISO image %s has been mounted to Partition %s.' %
+               (image_name, partition.name))
+
+
+def cmd_partition_unmount_iso(cmd_ctx, cpc_name, partition_name):
+
+    client = zhmcclient.Client(cmd_ctx.session)
+    partition = find_partition(cmd_ctx, client, cpc_name, partition_name)
+
+    partition.pull_full_properties()
+    image_name = partition.get_property('boot-iso-image-name')
```

Comment:
The `boot-iso-image-name` property can very well be set, but the partition is set to boot from something else (e.g. mount ISO image, followed by property update of `boot-device` to boot from something else).

I suggest that we don't use just a non-empty `boot-iso-image-name` property as an indicator that boot from ISO is set, but in addition (!) that `boot-device=='iso-image'`.

Reply:
Withdrawing this comment, because we now use specific options to set/unset the boot-device property.